CHANGELOG.md

```diff
@@ -3,7 +3,64 @@
 All notable changes to this project will be documented in this file. This
 project adheres to [Semantic Versioning](http://semver.org/).
 
-## Unreleased
+## 0.51.0
```

Comment:
Previous releases are a bit more verbose. I'll write something up for the updates to monitoring, as I think we want to promote those.
Any other things to highlight? e.g. the `object.union_n` Wasm support?